ssl-config-generator/index.html

```diff
@@ -124,6 +124,41 @@
 </pre>
     </script>
 
+    <script id="lighttpd-template" type="text/x-handlebars-template">
+<h2>{{server}} {{serverVersion}} | {{securityProfile}} profile | OpenSSL {{opensslVersion}} | <a href="?{{queryString}}">link</a></h2>
+<p>Oldest compatible clients : {{clientList}}</p>
+<pre>
+$SERVER["socket"] == ":443" {
+    protocol     = "https://"
+    ssl.engine   = "enable"
+    ssl.disable-client-renegotiation = "enable"
+
+    # pemfile is cert+privkey, ca-file is the intermediate chain in one file
+    ssl.pemfile               = "/path/to/signed_cert_plus_private_key.pem"
+    ssl.ca-file               = "/path/to/intermediate_certificate.pem"
+    # for DH/DHE ciphers, dhparam should be >= 2048-bit
+    ssl.dh-file               = "/path/to/dhparam.pem" 
+    # ECDH/ECDHE ciphers curve strength (see `openssl ecparam -list_curves`)
+    ssl.ec-curve              = "secp384r1"
```

Comment:
After further discussions, it appears that p384 is the way to go, so you should leave that in place.